fuzz/asn1parse.c

```diff
@@ -34,12 +29,11 @@ int FuzzerInitialize(int *argc, char ***argv)
 
 int FuzzerTestOneInput(const uint8_t *buf, size_t len)
 {
-    (void)ASN1_parse_dump(bio_out, buf, len, 0, 0);
+    (void)ASN1_parse_dump(NULL /* ignore output */, buf, len, 0, 0);
```

Comment:
I do not think this change is right as then the fuzzing tests the code with a condition that will never used in practice. It seems that the "quick hack" to use BIO_s_mem() as fallback somehow irritates you but I do not see a problem with it. And if you have a problem with the BIO_new_file() on Windows, then feel free to add some #ifdef for that.

Reply:
A further alternative, which would be portable without the need for `#ifdef`, would be to have just
```
    bio_out = BIO_new(BIO_s_mem()); /* yet output will be ignored */
```
for all architectures.

Reply:
So will go for that.

Reply:
BTW, how about generally adding a "dev-null" BIO?
Or do we have that already in some form?

Reply:
> BTW, how about generally adding a "dev-null" BIO?
> Or do we have that already in some form?

It might be useful. On the other hand, please do not add it now. Post 3.0

Reply:
So I'll open a feature request. How to name that "sink" BIO?

Reply:
Heh, we actually have it already. BIO_s_null(). Not sure why I did not notice it before.

Reply:
Nice - so okay if we use it here instead of the mem BIO (htough you already approved?)

Reply:
Sure. Take it as approved with that change.